packages/mermaid/src/utils/getSubGraphTitleMargins.ts

```diff
@@ -0,0 +1,17 @@
+import { getConfig } from '../diagram-api/diagramAPI.js';
+
+export const getSubGraphTitleMargins = (): {
+  subGraphTitleTopMargin: number;
+  subGraphTitleBottomMargin: number;
+  subGraphTitleTotalMargin: number;
+} => {
+  const subGraphTitleTopMargin = getConfig().flowchart?.subGraphTitleMargin?.top || 0;
+  const subGraphTitleBottomMargin = getConfig().flowchart?.subGraphTitleMargin?.bottom || 0;
```

Comment:
For consistency, we always use `??` instead of `||` for default fallbacks as `||` could lead to bugs in certain cases.

```suggestion
  const subGraphTitleTopMargin = getConfig().flowchart?.subGraphTitleMargin?.top ?? 0;
  const subGraphTitleBottomMargin = getConfig().flowchart?.subGraphTitleMargin?.bottom ?? 0;
```

Reply:
Can you also remove the duplicate call to getConfig()?

```suggestion
  const { flowchart } = getConfig();
  const subGraphTitleTopMargin = flowchart?.subGraphTitleMargin?.top ?? 0;
  const subGraphTitleBottomMargin = flowchart?.subGraphTitleMargin?.bottom ?? 0;
```

Or even better, make config a parameter to the function, the config is already available from your callsites, so we can pass it in here, eliminating the call altogether.

Reply:
I added the parameter as a destructered `{ flowchart }` that will come if passing `getConfig()` as argument when calling the function.